context/app/static/js/components/entity-search/searchkit-modifications/useSearchkitSDK.js

```diff
@@ -5,7 +5,7 @@ import useDeepCompareEffect from 'use-deep-compare-effect';
 // Copied from https://github.com/searchkit/searchkit/blob/6d11b204520009a705fe207535bd4f18d083d361/packages/searchkit-sdk/src/react-hooks/index.ts
 // Modified to handle initial filters and use our custom transformer
 
-const useSearchkitSDK = (config, variables, transformer, filters) => {
+const useSearchkitSDK = (config, variables, transformer, filters, defaultSort) => {
```

Comment:
```suggestion
const useSearchkitSDK = ({config, variables, transformer, filters, defaultSort}) => {
```
The parameter list is getting long enough that it would probably be good to destructure.

Reply:
I'll pull all of this into a single hook and refactor in a follow-up PR, thanks!